WordPress/src/test/java/org/wordpress/android/ui/reader/viewmodels/ReaderPostDetailViewModelTest.kt

```diff
@@ -77,17 +89,18 @@ class ReaderPostDetailViewModelTest {
     @Mock private lateinit var reblogUseCase: ReblogUseCase
     @Mock private lateinit var readerFetchRelatedPostsUseCase: ReaderFetchRelatedPostsUseCase
     @Mock private lateinit var eventBusWrapper: EventBusWrapper
+    @Mock private lateinit var readerSimplePost: ReaderSimplePost
+    @Mock private lateinit var analyticsUtilsWrapper: AnalyticsUtilsWrapper
 
     private val fakePostFollowStatusChangedFeed = MutableLiveData<FollowStatusChanged>()
     private val fakeRefreshPostFeed = MutableLiveData<Event<Unit>>()
     private val fakeNavigationFeed = MutableLiveData<Event<ReaderNavigationEvents>>()
     private val fakeSnackBarFeed = MutableLiveData<Event<SnackbarMessageHolder>>()
 
     private val readerPost = createDummyReaderPost(2)
-    private val readerSimplePost = ReaderSimplePost()
 
-    private val localRelatedPosts = ReaderSimplePostList().apply { add(readerSimplePost) }
-    private val globalRelatedPosts = ReaderSimplePostList().apply { add(readerSimplePost) }
+    private lateinit var localRelatedPosts: ReaderSimplePostList
+    private lateinit var globalRelatedPosts: ReaderSimplePostList
```

Comment:
Suggestion (💡): Consider merging those two into one `relatedPosts` filed. Currently there is no value in having them separate, is it?

Reply:
Yes we can merge it, in 19baace.